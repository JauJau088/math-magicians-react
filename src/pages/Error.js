import React from 'react';

const Error = () => (
  <div className="SectionContainer">
    <div className="SectionContentContainer">
      <div className="SectionContent">
        404: Page not found!
      </div>
    </div>
  </div>
);

export default Error;
