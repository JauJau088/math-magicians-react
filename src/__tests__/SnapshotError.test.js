import React from 'react';
import renderer from 'react-test-renderer';
import Error from '../pages/Error';

describe('Snapshot test for Error page', () => {
  it('It should render an identical template as the snapshot', () => {
    const ErrorPage = renderer.create(<Error />).toJSON();
    expect(ErrorPage).toMatchSnapshot();
  });
});
