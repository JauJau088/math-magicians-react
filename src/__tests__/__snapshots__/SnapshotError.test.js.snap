// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Error page It should render an identical template as the snapshot 1`] = `
<div
  className="SectionContainer"
>
  <div
    className="ChalkBoard SectionContentContainer"
  >
    <div
      className="SectionContent"
    >
      404: Page not found!
    </div>
  </div>
</div>
`;
